src/managers/default/index.js

```diff
@@ -280,8 +280,6 @@ class DefaultViewManager {
 
 			if(left < this.container.scrollWidth) {
 				this.scrollBy(this.layout.delta, 0);
-			} else if (left - this.layout.columnWidth === this.container.scrollWidth) {
-				this.scrollTo(this.container.scrollWidth - this.layout.delta, 0);
```

Comment:
I think we still need to scroll here, but also call:
`next = this.views.last().section.next();`